build-ps/jenkins/test-binary.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+#
+# Execute this tool to test binary releases
+#
+#  possible jenkins vars:
+#      BUILD_TYPE = (release debug release-asan debug-asan valgrind)
+#      DEFAULT_TESTING = (yes no)
+#      HOTBACKUP_TESTING = (yes no)
+#      TOKUDB_ENGINES_MTR = (yes no)
+#      MTR_REPEAT
+#      MTR_ARGS
+#      MTR_ARGS_CTEST
+
+set -o errexit
+set -o xtrace
+
+WORKDIR_ABS=$(cd ${1:-.}; pwd -P)
+tar -C $WORKDIR_ABS -zxpf $(ls $WORKDIR_ABS/*.tar.gz | head -1)
+cd $WORKDIR_ABS/Percona-Server-*/mysql-test
+
+TOKUDB_PLUGIN=$(find $WORKDIR_ABS -type f -name 'ha_tokudb.so')
+HOTBACKUP_LIB=$(find $WORKDIR_ABS -type f -name 'libHotBackup.so')
+HOTBACKUP_PLUGIN=$(find $WORKDIR_ABS -type f -name 'tokudb_backup.so')
+
+if [ "x${BUILD_TYPE}" = "xdebug" -o "x${BUILD_TYPE}" = "xvalgrind" -o "x${BUILD_TYPE}" = "xdebug-asan" ]; then
+    MTR_ARGS+=" --debug-server"
+fi
+if [ "x${BUILD_TYPE}" = "xvalgrind" ]; then
+    MTR_ARGS+=" --valgrind --valgrind-clients"
+elif [ "x${BUILD_TYPE}" = "xdebug-asan" -o "x${BUILD_TYPE}" = "xrelease-asan" ]; then
+    export ASAN_OPTIONS=allocator_may_return_null=true
+else
+    EATMYDATA=$(find /lib* /usr/lib* /usr/local/lib* -type f -name '*eatmyda*.so*' | head -n1)
+    if [ "x${EATMYDATA}" = "x" ]; then
+        echo "No libeatmydata.so lib found"
+        exit 1
+    fi
+fi
+
+if [ "x${MTR_REPEAT}" != "x" ]; then
+    MTR_ARGS+=" --repeat=${MTR_REPEAT}"
+fi
+
+MTR_ARGS_CTEST+=" --unit-tests-report"
+MTR_ARGS+=" --nounit-tests"
+
+if [ -f /etc/mecabrc ]; then
+    MTR_ARGS+=" --mysqld=--loose-mecab-rc-file=/etc/mecabrc"
+fi
+
+# Workaround http://bugs.mysql.com/bug.php?id=73280
+export OPENSSL_ENABLE_MD5_VERIFY=yes
```

Comment:
This may be workarounds for very old (currently unsupported) platforms. Consider removing.

Reply:
removed